PWGHF/Tasks/correlateEstimatorDnDeta.cxx

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file correlateEstimatorDnDeta.cxx
+/// \brief task for correlating the multiplicity estimator with generated dN/deta
+///
+/// \author Fabrizio Chinu <fabrizio.chinu@cern.ch>, Università and INFN Torino
+
+#include <string>
+#include <vector>
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+#include "Framework/StaticFor.h"
+#include "Common/DataModel/Multiplicity.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct CorrelateEstimatorDnDeta {
+  HistogramRegistry registry{"registry", {}};
+  static constexpr int8_t nEstimators = 8;
+  static constexpr std::string estimatorsNames[nEstimators] = {"FV0A", "FT0A", "FT0C", "FT0M", "FDDA", "FDDC", "FDDM", "NTPV"};
+
+  std::vector<unsigned> consideredParticles = {
+    11,  // e
+    13,  // mu
+    211, // pi
+    321, // K
+    2122 // p
```

Comment:
Does it break anything if you use the named codes here?

Reply:
No, it makes the code even more readable; thanks for the suggestion!